### DISCUSSION TITLE
CarpenPi

### DISCUSSION THREAD

**Discussion by jsteyn** [carpentriesoffline/CarpentriesOffline.github.io#3] [Team Posts]
Hi All

After our success with CarpenPi at the SSI CW, I would not like the project to lose momentum. I am very much in love with the project and as you all know I have a gaggle of Pis here in my house and am always looking for things to do with them. I have also turned into a bit of a Carpentries fanatic since the beginning of 2020 which means this project is right up my alley. Given this, I'd really like to push this project forward and would be happy to take the lead on this. I would like to know if you are happy with me taking on this role in the project. If you would be interested in staying involved and contributing to this project then please let me know.

Kind regards
Jannetta

**Comment by abhidg** [comment]
@jsteyn That's great! I'll try to find some time to work on it as well -- this is an awesome project. Maybe it'd be good to set up meetings once a month online (at the very least) to keep the momentum going.

**Comment by FlicAnderson** [comment]
I agree, @jsteyn, I'm happy to keep working on this and happy for you to coordinate and lead :)  

I think the suggestion @abhidg made about monthly meetings was a good one, maybe we need a doodle poll for dates for the 'inaugural CarpenPi dev meeting'? 😁

**Comment by DimmestP** [comment]
Hey all,

Yep, I am more than happy to contribute which every way I can! Setting up a monthly meeting sounds great.

**Comment by DimmestP** [comment]
Hi @jsteyn, 
Any updates on meetings/work?

Yours,
Sam
